subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,147 @@
+- <<sec:console_build_result,The build result>>: Are there other interesting outcomes e.g. tasks that have been skipped or were up-to-date?
+- <<sec:console_build_output,The build output>>: Is there anything I should know about my build right now e.g. tests have failed and emitted warnings?
+
+[[sec:console_build_progress_bar]]
+==== Build progress bar
+
+The build progress bar gives the user a very fast way of telling whether the build will be finished soon. As the build performs work, the progress indicator, the `=` character, will move from left to right. At any given time the build progress bar clearly renders the currently executed phase of the build lifecycle (see xref:build_phases[]) as well as the time spent during that phase.
+
+The following examples shows the progress bar during the initializing, configuration and execution phase of the build lifecycle:
+
+----
+<-------------> 0% INITIALIZING [2s]
+<==-----------> 25% CONFIGURING [4s]
+<=========----> 64% EXECUTING [17s]
+----
+
+[[sec:console_work_in_progress_display]]
+==== Work in-progress display
+
+The console provides a more fined-grained view of the actual work being performed. This information can be found directly underneath the build progress bar. Each line represents a worker thread or process that can perform work in parallel e.g. resolving dependencies, executing a task action. If an available worker is not utilized then it is marked with `IDLE`. The number of available workers equates to the number of processors on the machine executing the build. If needed a user can provide the number of workers through the <<gradle_command_line,command line>>  option `--max-workers`.
+
+The following portion of the console output shows the work in-progress display in action:
+
+----
+<==========---> 77% EXECUTING [10s]
+> :codeQuality:classpathManifest > Resolve dependencies :codeQuality:runtimeClasspath
+> :ivy:classpathManifest > Resolve dependencies :ivy:runtimeClasspath
+> IDLE
+> :antlr:classpathManifest > Resolve dependencies :antlr:runtimeClasspath
+> :scala:compileJava > Resolve dependencies :scala:compileClasspath
+> :buildInit:classpathManifest > Resolve dependencies :buildInit:runtimeClasspath
+> :jacoco:classpathManifest > Resolve dependencies :jacoco:runtimeClasspath
+> IDLE
+----
+
+[[sec:console_build_result]]
+==== Build result
+
+After finishing the build, the console will render the outcome of the build and a ratio of work performed vs. work avoided. The outcome indicates if the build was successful or failed. It also displays the elapsed time it took to run the overall build. The work ratio gives the user an idea about the number of performed tasks or tasks that did not need to be executed. The intention of the summarized view is to avoid too much information in the console output during the build.
+
+----
+BUILD SUCCESSFUL in 2m 10s
+411 actionable tasks: 381 executed, 30 up-to-date
+----
+
+[[sec:console_build_output]]
+==== Build output
```

Comment:
I think this section should follow the order that the parts of the console appear on the screen, so it should be earlier than all of the others if someone was reading this from top to bottom.